deps/wazuh_testing/wazuh_testing/qa_ctl/run_tests/Test.py

```diff
@@ -2,16 +2,16 @@
 
 
 class Test(ABC):
-    """ Abstract class to be extended by Pytest and used by TestLauncher.
+    """Abstract class to be extended by Pytest and used by TestLauncher.
 
     Attributes:
         tests_path (str): Path to the set of tests to be executed
         tests_run_dir (str): Path to the directory from where the tests are going to be executed
-
+        tests_result_path(str): Path to the directory where the reports will be stored in the local machine
```

Comment:
```suggestion
        tests_result_path(str): Path to the directory where the reports will be stored in the local machine
        
```

Reply:
Done in [commit](https://github.com/wazuh/wazuh-qa/pull/1714/commits/027dae406b0ff5b1afa971466cc5c9a1062d85ae)